tests/test_models.py

```diff
@@ -791,3 +791,49 @@ def test_fileinstance_validation(app, db, dummy_location):
     f = FileInstance.create()
     f.set_uri('x' * 255, 1000, 1000)  # Should not raise
     pytest.raises(ValueError, f.set_uri, 'x' * 256, 1000, 1000)
+
+
+def test_object_version_tags(app, db, dummy_location):
+    """Test object version tags."""
+    f = FileInstance(uri="f1", size=1, checksum="mychecksum")
+    db.session.add(f)
+    db.session.commit()
+    b = Bucket.create()
+    obj1 = ObjectVersion.create(b, "test").set_file(f)
+    ObjectVersionTag.create(obj1, "mykey", "testvalue")
+    ObjectVersionTag.create(obj1, "another_key", "another value")
+    db.session.commit()
+
+    # Duplicate key
+    pytest.raises(
+        Exception, ObjectVersionTag.create, obj1, "mykey", "newvalue")
```

Comment:
Can the exception be made more specific - e.g. is it ``IntegrityError`` being thrown? Would you mind also fixing it in the BucketTags test.